### ISSUE TITLE
[FEATURE] Keep Selection-Highlight Centered During Switching

### ISSUE THREAD

**Issue by xSeekTruthx** [G-dH/advanced-alttab-window-switcher#115]
**Is your feature request related to a problem? Please describe.**

While switching: The highlighted window selector goes to the right most of the selection screen. With multiple windows, this results in the user not knowing what window will appear next.

**Describe the solution you'd like**

Keep the highlighted selection centered, so when switching between multiple windows, it can be done very fast without having to shift back once the user accidentally skips the desired window.

**Additional context**

Love this extension!

**Comment by G-dH** [comment]
Thanks for your feedback!

>I think most of us have at least 10+ windows open at all times? Or is that just me? 😆

That's why AATWS got the search feature, you just press 2-3 letters and the window is in focus ;)

**Comment by xSeekTruthx** [comment]
> Thanks for your feedback!
> 
> > I think most of us have at least 10+ windows open at all times? Or is that just me? 😆
> 
> That's why AATWS got the search feature, you just press 2-3 letters and the window is in focus ;)



> Thanks for your feedback!
> 
> > I think most of us have at least 10+ windows open at all times? Or is that just me? 😆
> 
> That's why AATWS got the search feature, you just press 2-3 letters and the window is in focus ;)

Indeed, however, I do not know how what word is required when most of these windows are in the same app with 5+ tabs each. Alt-Tab will just get me there... 

Why am I only able to comment if I close it? Please have this remain open.

**Comment by G-dH** [comment]
I would rather use paging instead, because everything moving with each press of the Tab key is a nightmare for me.

**Comment by xSeekTruthx** [comment]
> I would rather use paging instead, because everything moving with each press of the Tab key is a nightmare for me.

Understood. I'm not a developer, just a UX guy.

**Comment by G-dH** [comment]
I'm talking about UX.

**Comment by xSeekTruthx** [comment]
> I'm talking about UX.

Oh ok, I thought you were saying it was a nightmare to code it to stay centered.

**Comment by xSeekTruthx** [comment]
Here's a better example of why, continuity across resolutions:

**1920x1080**

![screenshotOG](https://github.com/user-attachments/assets/47aa72e7-d805-4329-a822-e7efbb9af5ce)

**1600x900**

![screenshotOG1600900](https://github.com/user-attachments/assets/dde5e08c-eced-4ab5-a8aa-8eece80f01b9)

**1360x768** I'm 9ft away on a 32in TV :smile: 

![screenshotOG1360768](https://github.com/user-attachments/assets/091f5bc2-1f52-4612-ac5f-7cc0075e9661)

I use 1280x720 at times in different situations. A switcher is important to large communities that require these lower resolutions on varying devices. :heart_hands:

**Comment by xSeekTruthx** [comment]
![screenshotOGreality](https://github.com/user-attachments/assets/3d89fdf7-f417-46c9-9fe0-807d4e8b812f)

**Comment by G-dH** [comment]
I understand the issue, I just want to fix it without everything constantly moving ;)

**Comment by xSeekTruthx** [comment]
You could just make it an optional feature 🍻
